controllers/storagecluster/external_resources.go

```diff
@@ -375,7 +375,7 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 				enableRookCSICephFS = true
 			} else if d.Name == cephRbdStorageClassName {
 				scc = newCephBlockPoolStorageClassConfiguration(instance)
-			} else if d.Name == cephRbdRadosNamespaceStorageClassName {
+			} else if strings.Contains(d.Name, cephRbdRadosNamespaceStorageClassName) { // ceph-rbd-rados-namespace-<radosNamespaceName>
```

Comment:
using contains might be a bad idea isn't it? what if I have 2 Radosnamespace with same name but one with different prefix?

Reply:
So this prefix will be constant, and name should be always different/unique,
We will be passing the storageclass name from the python script, https://github.com/rook/rook/pull/15243
And this check is to just make sure it class this `else if` case if its has the prefix name.

Reply:
instead of contains use HasPrefix you want to check for specific prefix?